src/modules/common/containers/__tests__/Error.js

```diff
@@ -3,30 +3,35 @@ import { Provider } from 'react-redux'
 import renderer from 'react-test-renderer'
```

Comment:
please rename this file to Error.spec.js

Reply:
Will be done in common module task